src/scripts/temporaryBattle/TemporaryBattleList.ts

```diff
@@ -231,7 +231,7 @@ TemporaryBattleList['Suicune 6'] = new TemporaryBattle(
 TemporaryBattleList['Spiky-eared Pichu'] = new TemporaryBattle(
     'Spiky-eared Pichu',
     [new GymPokemon('Spiky-eared Pichu', 3178500, 20)],
-    '',
+    '<b><i>You caught the Spiky-eared Pichu!</i></b>',
```

Comment:
It's defeated message will now show up (before the window just went away).
So we now need to add an image for the pichu, so the modal has something to show

Reply:
I used the directory to not have to add another image in the temporaryBattle folder. If we like this solution I can clean up some of the other potentially unnecessary image files in the temporaryBattle folder to save up a tiny bit of space using the same method in a different PR.